framework/test/mocha/common/lisk-mocha-runner/index.js

```diff
@@ -142,8 +142,6 @@ if (testPathPattern && testPathPattern.indexOf('-') === 0) {
 	testPathPattern = null;
 }
 
-mochaCliOptions.unshift('--', '--opts', 'test/mocha/mocha.opts');
```

Comment:
❤️

Reply:
I suspect this is causing https://github.com/LiskHQ/lisk-sdk/issues/4124